support-payment-api/build.sbt

```diff
@@ -53,6 +53,8 @@ libraryDependencies ++= Seq(
   "org.scala-lang.modules" %% "scala-xml" % "1.2.0"
 )
 
+dependencyOverrides += "com.fasterxml.jackson.core" % "jackson-databind" % jacksonDatabindVersion
```

Comment:
not sure why this has happened in this branch, but without this we get the following error:
`com.fasterxml.jackson.databind.JsonMappingException: Scala module 2.11.4 requires Jackson Databind version >= 2.11.0 and < 2.12.0`

Reply:
that is very annoying - there seems to be a one way restriction and so noawadays you have to force an earlier version when play 2.8 and AWS SDK 2 are involved.